src/gridtools/clang/storage_dsl.hpp

```diff
@@ -24,150 +24,146 @@
-
-            storage &operator()(dimension, direction);
-            storage &operator()(direction, dimension);
-            storage &operator()(direction, direction);
-            /** @} */
-
-            /**
-             * @name 3D access
-             * @{
-             */
-            storage &operator()(dimension, dimension, dimension);
-
-            storage &operator()(direction, direction, direction);
-            storage &operator()(dimension, direction, direction);
-            storage &operator()(direction, dimension, direction);
-            storage &operator()(direction, direction, dimension);
-            storage &operator()(dimension, dimension, direction);
-            storage &operator()(dimension, direction, dimension);
-            storage &operator()(direction, dimension, dimension);
-            /** @} */
-
-            operator double() const;
-        };
-
-        /**
-         * @brief Dummy 3-dimensional temporary storage
-         * @ingroup gridtools_clang
-         */
-        struct temporary_storage {
-            temporary_storage();
-
-            template < typename T >
-            temporary_storage(T...);
-
-            temporary_storage &operator=(const temporary_storage &);
-            temporary_storage &operator+=(const temporary_storage &);
-            temporary_storage &operator-=(const temporary_storage &);
-            temporary_storage &operator/=(const temporary_storage &);
-            temporary_storage &operator*=(const temporary_storage &);
-
-            /**
-             * @name 1D access
-             * @{
-             */
-            temporary_storage &operator()(direction);
-            temporary_storage &operator()(dimension);
-            temporary_storage &operator()(offset);
-            /** @} */
-
-            /**
-             * @name 2D access
-             * @{
-             */
-            temporary_storage &operator()(dimension, dimension);
-
-            temporary_storage &operator()(dimension, direction);
-            temporary_storage &operator()(direction, dimension);
-            temporary_storage &operator()(direction, direction);
-            /** @} */
-
-            /**
-             * @name 3D access
-             * @{
-             */
-            temporary_storage &operator()(dimension, dimension, dimension);
-
-            temporary_storage &operator()(direction, direction, direction);
-            temporary_storage &operator()(dimension, direction, direction);
-            temporary_storage &operator()(direction, dimension, direction);
-            temporary_storage &operator()(direction, direction, dimension);
-            temporary_storage &operator()(dimension, dimension, direction);
-            temporary_storage &operator()(dimension, direction, dimension);
-            temporary_storage &operator()(direction, dimension, dimension);
-            /** @} */
-
-            operator double() const;
-        };
+namespace clang {

```

Comment:
at the top there is a namespace gridtools. I think it shouldnt be here since this is a new language that has no relation with GT. But no need to change it in this PR. Maybe better create an issue and we can tackle this later